diff --git a/build/reactNativeConfig/config.js b/build/reactNativeConfig/config.js
index 65ef8980f00d2a6e071021e61c5de85cb2c00358..da060c6553c3ef93ff9772551ebabf112bf0e8e3 100644
--- a/build/reactNativeConfig/config.js
+++ b/build/reactNativeConfig/config.js
@@ -53,13 +53,20 @@ exports.loadConfigAsync = loadConfigAsync;
  */
 function requireConfig(filepath, configContents) {
     try {
-        const config = (0, require_from_string_1.default)(configContents, filepath, {
-            prependPaths: [mockedNativeModules],
-        });
+        delete require.cache[filepath];
+        const config = require(filepath);
         return config.default ?? config ?? null;
     }
     catch {
-        return null;
+        try {
+            const config = (0, require_from_string_1.default)(configContents, filepath, {
+                prependPaths: [mockedNativeModules],
+            });
+            return config.default ?? config ?? null;
+        }
+        catch {
+            return null;
+        }
     }
 }
 //# sourceMappingURL=config.js.map
